flake.nix

```diff
@@ -7,28 +7,27 @@
     ];
   };
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
     flake-utils.url = "github:numtide/flake-utils";
-    nix2container.url = "github:nlewo/nix2container";
-    nix2container.inputs.nixpkgs.follows = "nixpkgs";
-    nix2container.inputs.flake-utils.follows = "flake-utils";
-    nix-editor.url = "github:snowfallorg/nix-editor";
-    nix-editor.inputs.utils.follows = "flake-utils";
+    git-hooks.inputs.nixpkgs.follows = "nixpkgs";
+    git-hooks.url = "github:cachix/git-hooks.nix";
     nix-editor.inputs.nixpkgs.follows = "nixpkgs";
-    rust-overlay.url = "github:oxalica/rust-overlay";
-    rust-overlay.inputs.nixpkgs.follows = "nixpkgs";
-    nix-fast-build.url = "github:Mic92/nix-fast-build";
+    nix-editor.inputs.utils.follows = "flake-utils";
+    nix-editor.url = "github:snowfallorg/nix-editor";
+    nix-eval-jobs.url = "github:nix-community/nix-eval-jobs";
     nix-fast-build.inputs.flake-parts.follows = "flake-parts";
     nix-fast-build.inputs.nixpkgs.follows = "nixpkgs";
     nix-fast-build.inputs.treefmt-nix.follows = "treefmt-nix";
-    flake-parts.url = "github:hercules-ci/flake-parts";
-    treefmt-nix.url = "github:numtide/treefmt-nix";
-    treefmt-nix.inputs.nixpkgs.follows = "nixpkgs";
-    git-hooks.url = "github:cachix/git-hooks.nix";
-    git-hooks.inputs.nixpkgs.follows = "nixpkgs";
+    nix-fast-build.url = "github:Mic92/nix-fast-build";
+    nix2container.inputs.nixpkgs.follows = "nixpkgs";
+    nix2container.url = "github:nlewo/nix2container";
     nixpkgs-go124.url = "github:Nixos/nixpkgs/d2ac4dfa61fba987a84a0a81555da57ae0b9a2b0";
-    nixpkgs-pgbackrest.url = "github:nixos/nixpkgs/nixos-unstable-small";
-    nix-eval-jobs.url = "github:nix-community/nix-eval-jobs";
+    nixpkgs-oldstable.url = "github:NixOS/nixpkgs/a76c4553d7e741e17f289224eda135423de0491d";
```

Comment:
We might want to add a comment here above to give more context about that version. Should we update it in the future ?